storage-proofs/src/parameter_cache.rs

```diff
@@ -300,6 +303,7 @@ fn write_cached_verifying_key<E: JubjubEngine>(
     })
 }
 
+#[allow(dead_code)]
 fn write_cached_params<E: JubjubEngine>(
```

Comment:
why is this dead code, isn't this used above Oo

Reply:
Yes, was previously unused and I missed it after adding back in, thanks.